src/main/java/Duke.java

```diff
@@ -6,5 +8,55 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        printGreetings();
+        storeTasks();
+    }
+
+    public static void printGreetings() {
+        String greeting = "____________________________________________________________\n"
+                + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + "____________________________________________________________\n"
+                + "\n";
+
+        System.out.println(greeting);
+    }
+
+    public static void storeTasks() {
+        String line;
+        Task[] tasks = new Task[100];
+        Scanner in = new Scanner(System.in);
+        line  = in.nextLine();
+        int taskIndex = 0;
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                System.out.println("____________________________________________________________\n");
+                System.out.println("Here are the tasks in your list:\n");
+                for (int i = 1; i <= taskIndex; i++) {
+                    System.out.println(i + "." + "["+ tasks[i-1].getStatusIcon() + "] " + tasks[i-1].description + "\n");
+                }
+                System.out.println("____________________________________________________________\n");
+            } else if (line.startsWith("done")) {
+                int taskNo = Integer.parseInt(line.substring(5));
```

Comment:
Just a suggestion, maybe you can try accounting for cases where the taskNo is more than 1 digit. 
For example you can split the variable "line" into multiple substrings and then convert the second string to integer as follows:
String[] words = line.split(" ");
int taskNo = Integer.parseInt(words[1]);